cpk_lib_python_github/github_app_token_generator_package/github_app_token_generator/cli.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Command line interface for GitHub App Token Generator."""
+import argparse
+
+from colorama import Fore, Style
+
+
+def create_parser() -> argparse.ArgumentParser:
+    """Create and configure argument parser."""
+    parser = argparse.ArgumentParser(
+        description=(
+            f"{Fore.GREEN}{Style.BRIGHT}Generate and manage GitHub App "
+            f"installation tokens{Style.RESET_ALL}"
+        ),
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog=f"""
+{Fore.GREEN}{Style.BRIGHT}Examples:{Style.RESET_ALL}
+  {Fore.CYAN}# Generate tokens (using private key file){Style.RESET_ALL}
+  {Fore.YELLOW}%(prog)s{Style.RESET_ALL} {Fore.CYAN}--org{Style.RESET_ALL} myorg \\
+    {Fore.CYAN}--app-id{Style.RESET_ALL} APP_ID \\
+    {Fore.CYAN}--private-key-path{Style.RESET_ALL} /path/to/key.pem
+
+  {Fore.CYAN}# Generate tokens (using private key content){Style.RESET_ALL}
+  {Fore.YELLOW}%(prog)s{Style.RESET_ALL} {Fore.CYAN}--org{Style.RESET_ALL} myorg \\
+    {Fore.CYAN}--app-id{Style.RESET_ALL} APP_ID \\
+    {Fore.CYAN}--private-key{Style.RESET_ALL} "$(cat /path/to/key.pem)"
+
+{Fore.BLUE}{Style.BRIGHT}Environment Variables:{Style.RESET_ALL}
+  {Fore.MAGENTA}APP_ID{Style.RESET_ALL}                    GitHub App ID
+  {Fore.MAGENTA}PRIVATE_KEY_PATH{Style.RESET_ALL}          Path to private key file
+  {Fore.MAGENTA}PRIVATE_KEY{Style.RESET_ALL}               Private key content directly
+        """,
+    )
+
+    # Add arguments
+    parser.add_argument(
+        "--app-id", help="GitHub App ID (or set APP_ID env var)", metavar="ID"
+    )
```

Comment:
good idea but not important 

```suggestion
    parser.add_argument(
        "--app-id", type=int, help="GitHub App ID (or set APP_ID env var)", metavar="ID"
    )
```

Reply:
Addressed